<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasMethodDlg</class>
 <widget class="QDialog" name="MeasMethodDlg">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measurement Method Setting</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>570</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Measurement Method</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbMeasType">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Single electrode</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Double electrode</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="gbLinCheck">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>221</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Electrode Linearity Check  (in saline or cell)</string>
   </property>
   <widget class="QLineEdit" name="leLinMinCurrent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-1000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLinMaxCurrent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>+1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>I_min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>I_max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of injection levels</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test range (pA)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLinLevelNum">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="linCheckButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Check </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Average resistance (MOhm)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLinAverRes">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Standard deviation (MOhm)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLinResStd">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Results</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbGeneral">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>General settings</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output channel number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leGenInChannelNum">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input channel number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leGenOutChannelNum">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leGenInjLen">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Injection length (msec)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leGenSamplingRate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Desired sampling rate (Hz)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="aecTools">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>401</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>AEC tools</string>
   </property>
   <widget class="QGroupBox" name="gbCaliibration">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Calibration  (in cell)</string>
    </property>
    <widget class="QPushButton" name="calibrateButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibrate</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Injection amplitude (pA)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leCalInjAmp">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbTest">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Kernel Test  (in cell)</string>
    </property>
    <widget class="QComboBox" name="cbTestInputType">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>White noise</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Impulse</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Step</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square pulses</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Input type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTestSqWaveLength">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTestSqWaveLength">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Square pulse length (msec)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(-1: No control recordig)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTestCtrlChannelNum">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>-1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testKernelButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>75</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Kernel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Injection amplitude (pA)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTestInjAmp">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Control channel number </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbKernel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>151</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Kernel Upload (in cell)</string>
    </property>
    <widget class="QListWidget" name="lwKernelDisplay">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Electrode kernel file name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leKernelFileName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="loadKernelButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Kernel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lInfo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>641</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Data acquistion results</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Average sampling rate (Hz)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Min sampling rate (Hz)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAverSamplRate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMinTimeB2S">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Std in sampling rate (Hz)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSTDSamplingRate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDesPerMinRate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Desired / Min rate </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max sampling time (msec)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMaxSTime">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MeasMethodDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
